src/ThreeJsPanel.ts

```diff
@@ -429,6 +427,13 @@ export class ThreeJsPanel {
   }
 
   render(): void {
+    // update the axis helper in case the view was rotated
+    if (!isOrthographicCamera(this.camera)) {
+      this.axisHelperObject.rotation.setFromRotationMatrix(this.camera.matrixWorldInverse);
+    } else {
+      this.orthoScale = this.controls.scale;
+    }
+
```

Comment:
changes in this file fix issue with axis helper not moving when reset camera happens.